Geometry/GEMGeometry/src/ME0Chamber.cc

```diff
@@ -78,3 +78,23 @@ const ME0EtaPartition* ME0Chamber::etaPartition(int isl) const {
   }
   return 0;
 }
+
+float ME0Chamber::computeDeltaPhi(const LocalPoint& position, const LocalVector& direction ) const {
```

Comment:
(position phi - momentum phi) is a more complicated variable and is not as useful for track matching as ( position phi at end of ME0 - position phi at beginning of ME0 ). This is why we are moving to using it.

The reason is that the position one depends only on the precise measurement of phi position. The other one actually indirectly includes the eta measurement. This is because the strips are constant in phi only with respect to the origin. The angle in phi between the position and momentum of the segment in the chamber depends on the precise location of the segment's position in layer 1 and layer 6 in the eta direction. The difference in phi will be less if the segment is at the bottom of the strip at layer 1 and at the top of layer 6 than if it hit the center of both strips. Because the eta measurement is much less precise than the phi measurement this means that the matching between the pixel tracks and the segments is dominated by eta precision.